cypress/integration/checkin/checkin.basic.spec.js

```diff
@@ -0,0 +1,127 @@
+import moment from 'moment';
+import uuid from 'uuid';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import TestTypes from '../../support/dictionary/testTypes';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import checkoutActions from '../../support/fragments/checkout/checkout';
+import Users from '../../support/fragments/users/users';
+import InTransitModal from '../../support/fragments/checkin/modals/inTransit';
+import { Button, Pane, including, Modal } from '../../../interactors';
+
+describe('Check In - Actions ', () => {
+  const userData = {
+    permissions: [permissions.checkinAll.gui,
+      permissions.uiUsersViewLoans.gui,
+      permissions.uiUsersView.gui,
+      permissions.uiInventoryViewInstances.gui,
+      permissions.uiUsersfeefinesCRUD.gui],
+  };
+  const itemData = {
+    barcode: generateItemBarcode(),
+    instanceTitle: `Instance ${getRandomPostfix()}`,
+
+  };
+  const loanDetailsButton = Button('Loan details');
```

Comment:
try to find appropriate fragment to keep interactor interactions

Reply:
Is it  an appropriate way to hide the steps of test case into fragments so that the test won't be readable as every step in the testcase?

Reply:
may be it's good time to improve TC in TestRail too. Steps which contains just actions like click to some element or type something - bad pattern of test design. Test and test steps should be focused to business features and user actions.
For example: loanDetailsButton
You can add wrapper openLoanDetails and move into it these lines:

- cy.do(loanDetailsButton.click());
- cy.expect(Pane(including(userProperties.username)).exists());
- cy.expect(Pane(including('Loan details')).exists());

You will get more clear test steps design, simpler code review expenses

Reply:
I have done changes. Thank you!